upgrade/util/util.go

```diff
@@ -1198,6 +1224,26 @@ func updateOrphans(namespace string, lhClient *lhclientset.Clientset, orphans ma
 	return nil
 }
 
+func updateSystemBackups(namespace string, lhClient *lhclientset.Clientset, orphans map[string]*longhorn.SystemBackup) error {
+	existingSystemBackupList, err := lhClient.LonghornV1beta2().SystemBackups(namespace).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	for _, existingSystemBackup := range existingSystemBackupList.Items {
+		sb, ok := orphans[existingSystemBackup.Name]
```

Comment:
I assume this should be caught if you create a system backup in an older version and upgrade to v1.8.0.
Again, please remember to test the functionality first and make sure it works correctly.

Reply:
Modified. Thanks.
I do tested the upgrade and it worked, I will test it again with new commit.

Reply:
For any deletion, make sure no valid backups will be deleted.